Python/other/find_Triplets_with_zero_sum.py

```diff
@@ -0,0 +1,88 @@
+'''
+  Author : Mohit Kumar
+  
+  python program to find triplets in a given  array whose sum is zero 
+
+'''
+
+# function to print triplets with 0 sum 
+
+def find_Triplets_with_zero_sum(arr, n): 
+   
+    ''' find triplets in a given  array whose sum is zero 
+        
+        Parameteres : 
+            arr : input array 
+            n = size of input array
+
+        Output :
+            if triplets found print their values 
+            else print No Triplet Found
+
+    '''
+   
+    found = False
+
+    # sort array elements 
+    arr.sort() 
+
+    for i in range(0, n - 1) : 
+    
+        # initialize left and right 
```

Comment:
Change all alphabets to meaningful variable names

Reply:
done